### DISCUSSION TITLE
Persistent volume claim

### DISCUSSION THREAD

**Discussion by purohitamit** [camunda-community-hub/community#154] [Q&A] [answered]
Hi All, I am new to camunda8, I had installed Camunda8 self managed in AKS using helm chart. I just need to ask that while installation there is a default elasticsearch pvc created which is of 64Gi. so is there any way to autoscale this pvc later depending on the usage. Thank you for any information or help.

**Comment by xomiamoore** [comment, marked as answer]
Hey @purohitamit, this GitHub Discussions is for questions related to the Camunda Community Hub and the community-led projects here -- in the future, [the Camunda forum](http://forum.camunda.io) is your best bet for usage questions.

To try and answer your question -- if you used the Helm chart at helm.camunda.io, it should be sufficient for 80% of use cases, but you can [adjust the default size to fit your environment](https://docs.camunda.io/docs/components/best-practices/architecture/sizing-your-environment/#camunda-8-self-managed). 

I encourage you to read a bit more about how scaling works on that same docs page. It's a little bit different than adding more Kubernetes pods, as that doesn't always add more throughput. That said, you can also look into [Kubernetes’ own autoscaler -- here’s the one for AKS.](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler/cloudprovider/azure)

**Reply by purohitamit** [reply]
Hi xomiamoore for the info. I will go through the docs again.
